teuthology/run.py

```diff
@@ -88,9 +88,10 @@ def fetch_tasks_if_needed(job_config):
     suite_sha1 = job_config.get('suite_sha1')
     suite_path = os.path.normpath(os.path.join(
         fetch_qa_suite(suite_branch, commit=suite_sha1),
-        job_config.get('suite_relpath', ''),
+        job_config.get('suite_relpath', 'qa'),
     ))
     sys.path.insert(1, suite_path)
+    import tasks
```

Comment:
This line is causing a test failure. I don't think we need it, since the actual importing of individual tasks is handled by the `run_tasks` module. Was it required in your testing?

Reply:
you're right, I thought it was necessary, but it was a ModuleNotFound error just from having the wrong path -- not including 'qa' -- by default